pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -135,10 +135,13 @@ const (
 	// Used in xds config. Metavalue bind to this key is used by pilot as xds server but not by envoy.
 	// So the meta data can be erased when pushing to envoy.
 	PilotMetaKey = "pilot_meta"
+
+	// TODO(yxue): separate h2c vs h2
+	H2Protocol = "h2"
```

Comment:
this isn't defined anywhere else?

Reply:
I didn't check but we need to refactor it in the near future once sniffing is enabled with mTLS. I have a TODO in the PR to merge `H2Protocol` with `applicationProtocols`